### PR TITLE
Fix readonly forms

### PR DESCRIPTION
**Pull request by vinetsky**

### CODE REVIEW

File: resources/styles/theme.css [hunk 1]
```diff
@@ -134,7 +134,8 @@ select {
 .editor.context_select.editor_disabled {
     background: var(--disabled-input-color);
     color: var(--disabled-text-color);
-    opacity: 0.7;
+    border-color: transparent;
+    border-left-width: 0;
```
Comment (stasguryev) [inline resources/styles/theme.css:138]: ?

File: resources/styles/theme.css [hunk 2]
```diff
@@ -465,6 +474,19 @@ td .checkbox.checkbox_switch i,
     overflow: hidden;
 }
 
+.editor_date-time.editor_readonly {
+    border-color: transparent;
+
```
Comment (stasguryev) [inline resources/styles/theme.css]: !

File: resources/styles/variables.css [hunk 3]
```diff
@@ -78,7 +78,7 @@
     --main-color: var(--theme-colors__primary);
     --selected-background-color: var(--theme-colors__main);
     --disabled-text-color: #444444;
-    --disabled-input-color: #b1b1b1;
+    --disabled-input-color: transparent;
```
Comment (stasguryev) [inline resources/styles/variables.css:81]: overrides in product

File: src/form/editors/impl/radioGroup/templates/radioGroup.hbs [hunk 4]
```diff
@@ -1 +1 @@
-<div class="js-container"></div>
\ No newline at end of file
+<div class="js-container editor_radio"></div>
```
Comment (stasguryev) [inline src/form/editors/impl/radioGroup/templates/radioGroup.hbs]: in className

File: resources/styles/variables.css (hunk 3 above)
Comment (vinetsky) [reply resources/styles/variables.css:81]: same values as in ThemeCSSWriter.cs

File: resources/styles/theme.css (hunk 2 above)
Comment (vinetsky) [reply resources/styles/theme.css]: убрал

File: src/form/editors/impl/radioGroup/templates/radioGroup.hbs (hunk 4 above)
Comment (vinetsky) [reply src/form/editors/impl/radioGroup/templates/radioGroup.hbs]: поправил

**Pull request merged**